source/features/avoid-accidental-submissions.tsx

```diff
@@ -1,40 +1,60 @@
+import React from 'dom-chef';
 import select from 'select-dom';
 import delegate from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import {isMac} from '../github-helpers';
```

Comment:
```suggestion
import {isMac} from '../github-helpers';
import features from '.';
```

Reply:
I got the order from:
https://github.com/sindresorhus/refined-github/blob/afa8da1a27e80f6364a3e68ab4b4c5a7c1916d2e/source/features/reactions-avatars.tsx#L7-L9

Reply:
We always sort the word "from" as a tree

Reply:
I didn't know that. Thanks.